### DISCUSSION TITLE
Segfault in transport_write

### DISCUSSION THREAD

**Discussion by ditoe** [FreeRDP/FreeRDP#8067] [Show and tell]
Hi,
I'm currently digging into freerdp and we have now a coredump from a broken connection.
Our version is not the newest, but in the newest code there are the same functions.
The connection is broken and then transport->frontBio is null.
BIO_should_retry will then end up in a segfault.
![FreeRDP_Dump_transport_write](https://user-images.githubusercontent.com/31221724/178545093-d3822ebd-14da-48bf-8813-c52b0e325f62.PNG)

I currently don't have a patch, but maybe this is useful too?

**Comment by mfleisz** [comment]
Need more information to debug this (how to reproduce, used command line, what FreeRDP version/commit, etc.) - see the issue template over at https://github.com/FreeRDP/FreeRDP/issues/new?assignees=&labels=&template=bug_report.md for more information.
